.github/workflows/build_and_test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   # Build: build Spark and run the tests for specified modules.
   build:
     name: "Build modules: ${{ matrix.modules }} ${{ matrix.comment }} (JDK ${{ matrix.java }}, ${{ matrix.hadoop }}, ${{ matrix.hive }})"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
nit: how about leaving comments about the next LTS of ubuntu-20.04 here?

Reply:
IMO `latest` was a bit ambiguous, so the new one (`ubuntu-20.04`)  can make it easier to understand in which environment we run tests. Thanks, @dongjoon-hyun !